modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/consensus/block/BlockConsensusInput.scala

```diff
@@ -43,8 +45,9 @@ object BlockConsensusInput {
   implicit def showBlockConsensusInput: Show[BlockConsensusInput] = {
     case OwnRoundTrigger   => "OwnRoundTrigger"
     case InspectionTrigger => "InspectionTrigger"
-    case Proposal(roundId, senderId, _, _, txs, _) =>
-      s"Proposal(roundId=${roundId.value.toString.take(8)}, senderId=${senderId.value.value.take(8)} txsCount=${txs.size})"
+    case Proposal(roundId, senderId, _, _, txs, allowSpendTxs, _) =>
+      s"Proposal(roundId=${roundId.value.toString
+          .take(8)}, senderId=${senderId.value.value.take(8)} txsCount=${txs.size}, allowSpendTxsCount=${allowSpendTxs.size})"
```

Comment:
`.take(8)` appears twice -- should this instead call a function with a descriptive name?

Reply:
Well, if we're going to reuse it, we can use .short or something on the Id. Here, I don't care about the duplication because making it generic, I'll basically remove this PR and make Transaction's logic generic

Reply:
Sounds fine. I was primarly concerned about the magic "8".